README.md

```diff
@@ -155,43 +163,42 @@ The name for each type can be whatever the user wants, with the two following ru
 Remember that the system handle will map each `/` for `__`, as mentioned in the configuration section,
 to allow an easy connection with ROS2 types.
 
-Each of the members of the dynamic type can be defined just by its type and name, such as `int32: "my_integer"`.
-
-*eProsima*'s *dtparser* supports the following basic types:
-
-- boolean
-- char8
-- char16
-- byte
-- int16
-- int32
-- int64
-- uint16
-- uint32
-- uint64
-- float32
-- float64
-- float128
-- string
-
-The dynamic types parser also allow nested structures.
-To create a nested structure, there must be a definition of the basic structure and a reference to the basic structure
-in the member of the nested structure, using the name of the simple structure.
+Currently, the `xtypes::idl::Parser` (or xtypes) doesn't support the following IDL 4.2 characteristics:
+
+- Any
+- Interfaces
+- Value Types
+- CORBA related features
+- Components related features
+- CCM related features
+- Template Modules
+- Extended Data-Types
+- Anonymous Types
+- Annotations
+
+For more details about IDL definition, please refer to [IDL documentation](https://www.omg.org/spec/IDL/4.2/PDF).
 
 The following is an example of a full configuration file that uses the ROS2 nested type
```

Comment:
This sample should be change. If the type comes from ros2, you can use a `types-from: ros2` to get it. It is no longer necessary to create

Reply:
As we talked, the example modification is on your side :).